docs/fastdds/statistics/dds_layer/domainParticipant.rst

```diff
@@ -121,5 +135,22 @@ The following examples show how to use all the previous methods:
 
         HISTORY_LATENCY_TOPIC;NETWORK_LATENCY_TOPIC;PUBLICATION_THROUGHPUT_TOPIC;SUBSCRIPTION_THROUGHPUT_TOPIC;RTPS_SENT_TOPIC;RTPS_LOST_TOPIC;HEARTBEAT_COUNT_TOPIC;ACKNACK_COUNT_TOPIC;NACKFRAG_COUNT_TOPIC;GAP_COUNT_TOPIC;DATA_COUNT_TOPIC;RESENT_DATAS_TOPIC;SAMPLE_DATAS_TOPIC;PDP_PACKETS_TOPIC;EDP_PACKETS_TOPIC;DISCOVERY_TOPIC;PHYSICAL_DATA_TOPIC
 
-Be aware that automatically enabling the statistics DataWriters using all these methods implies using the recommended
-QoS profile |STATISTICS_DATAWRITER_QOS-api|. For more information, please refer to :ref:`statistics_datawriter_qos`.
+.. note::
+
+    Be aware that automatically enabling the statistics DataWriters using all these methods implies using the recommended
+    QoS profile |STATISTICS_DATAWRITER_QOS-api|. For more information, please refer to :ref:`statistics_datawriter_qos`.
+    However, if an XML profile is defined, the QoS applied are those defined in the profile,
+    and for those QoS that are not specified in that profile, the default library QoS are applied
+    (see :ref:`dds_layer_publisher_dataWriterQos` for the standard eProsima's DataWriter QoS),
+    and not the recommended QoS for the Statistics DataWriters.
+
+For the creation of an automatically enabled Datawriter, the priority for setting its QoS is the following:
+
+* First, if a specific profile exists for the statistics topic, that one is applied.
+* If that is not the case but a generic profile for statistics DataWriters exists, that one is applied.
+* If no profile is defined in XML file, the recommended statistics QoS are applied.
+
+.. note::
+
+    The generic DataWriter profile defined in the `FASTRTPS_DEFAULT_PROFILES_FILE` XML needs to be named as
+    `GENERIC_STATISTICS_PROFILE`.
```

Comment:
Similarly, I suggest clarifying that the specific profile has to be named using the statistic topic alias (include a link to the API reference corresponding section) and, if confirmed that the topic name per se also works, indicate it so too.

Reply:
I have specified the need to name the profile with the topic alias. 
Now waiting for the decision for making official or not the use of the topic name instead of the alias.

Reply:
As decided, I have specified the behavior of this feature in 
61f9ae1d Refs #15014: Specify the behavior when enabling builtin statistic datawriters with specific QoS profile